src/import-data/ImportDataDestination.test.ts

```diff
@@ -104,17 +104,44 @@ describe('ImportDataDestination', () => {
     {
       importRequest: { type: 'pfb', url: new URL('https://service.prod.anvil.gi.ucsc.edu/path/to/file.pfb') },
       requiredAuthorizationDomain: 'test-auth-domain',
-      expectedArgs: { isProtectedData: true, requiredAuthorizationDomain: 'test-auth-domain' },
+      expectedArgs: {
+        cloudPlatform: undefined,
+        isProtectedData: true,
+        requiredAuthorizationDomain: 'test-auth-domain',
+      },
     },
     {
       importRequest: { type: 'pfb', url: new URL('https://example.com/path/to/file.pfb') },
       requiredAuthorizationDomain: undefined,
-      expectedArgs: { isProtectedData: false, requiredAuthorizationDomain: undefined },
+      expectedArgs: { cloudPlatform: undefined, isProtectedData: false, requiredAuthorizationDomain: undefined },
+    },
+    {
+      importRequest: {
+        type: 'tdr-snapshot-export',
+        manifestUrl: new URL('https://example.com/path/to/manifest.json'),
+        snapshot: {
+          id: '00001111-2222-3333-aaaa-bbbbccccdddd',
+          name: 'test-snapshot',
+          source: [
+            {
+              dataset: {
+                id: '00001111-2222-3333-aaaa-bbbbccccdddd',
+                name: 'test-dataset',
+                secureMonitoringEnabled: false,
+              },
+            },
+          ],
+          cloudPlatform: 'gcp',
+        },
+        syncPermissions: false,
+      },
+      requiredAuthorizationDomain: undefined,
+      expectedArgs: { cloudPlatform: 'GCP', isProtectedData: false, requiredAuthorizationDomain: undefined },
```

Comment:
is it worth also adding a test for `expectedArgs: { cloudPlatform: 'AZURE' …`? Either here or in `import-utils.test.ts` or both.